examples/SingleNeuron_HH1952/main.cpp

```diff
@@ -125,7 +128,7 @@ int main(int argc, char **argv) {
   engine::generate_neuron<HH_Neuron>();
 
   // integration period with stepsize
-  double t_start = 0.0, t_end = 100.0, t_stepsize = 0.05;
+  double t_start = 0.0, t_end = t_start + (insilico::engine::simulation_time), t_stepsize = 0.05;
```

Comment:
It will be great if you could add this consistency of explicitly mentioning times to all the existing examples.